RNAwesomeCardIO.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+package = JSON.parse(File.read('package.json'))
+
+Pod::Spec.new do |s|
+  s.name             = "RNAwesomeCardIO"
+  s.version          = package["version"]
+  s.summary          = package["description"]
+  s.requires_arc     = true
+  s.license          = package["license"]
+  s.homepage         = 'n/a'
+  s.authors          = { package["author"] => package["author"] }
+  s.source           = { :git => "https://github.com/Kerumen/react-native-awesome-card-io" }
```

Comment:
Similarly here,
```
s.source           = { :git => package["homepage"] }
```
Where you'd need to update `homepage` field in `package.json` to `https://github.com/Kerumen/react-native-awesome-card-io`.